changelog.d/2478.added.md

```diff
@@ -0,0 +1 @@
+Add support for Chunked Request Messages in HTTP Header Filter.
```

Comment:
This should be understandable to the user, so probably something like `Added support for intercepting streaming HTTP requests with an HTTP filter`